template/script.sh.erb

```diff
@@ -13,11 +13,14 @@ echo "TTT - $(date)"
 # Clean the environment
 module purge
 
+# Required modules
+module load xalt/latest <%= context.cuda_version %>
```

Comment:
I think cuda was being loaded in the kernels previously. Isn't _that_ where CUDA should be loaded because it's a completely separate environment?

Reply:
When we have the 'CUDA version' option in the form, I think it generally means CUDA should be available in all kernels.  In new kernels, we don't purge module and just append the required module so it is fine to load cuda in batch script.

Reply:
I'll take your word for it for now, but we should be sure it works when this lands in test. Should be some time in December, I'll keep you apprised.